__tests__/gsheets.js

```diff
@@ -0,0 +1,84 @@
+describe('src/gsheets', () => {
+  it('snapshot', () => {
+    const tickers = ['foo1', 'foo1'];
+    const ops = ['D', 'D'];
+    const qtties = [100, 100];
+    const totals = [100, 100];
+
+    let year = 2020;
+    let month = 2;
+    const dates = ['01/02/2020', '01/01/2020', , , , ,];
+
+    expect(dist.snapshot(year, month, dates, tickers, ops, qtties, totals))
+      .toMatchInlineSnapshot(`
+      Array [
+        Array [
+          "Ticker",
+          "Purchased qty.",
+          "Purchased total",
+          "Sold qty.",
+          "Sold total",
+          "Avg. price",
+        ],
+        Array [
+          "foo1",
+          200,
+          200,
+          0,
+          0,
+          1,
+        ],
+      ]
```

Comment:
I would separate this on per Array instead of per item, and only create a new line if it's another array. Despite turning those into enormous lines, I'll be clearer what's what because it'll resemble the table you're representing.

```suggestion
      Array [
        Array [ "Ticker", "Purchased qty.", "Purchased total", "Sold qty.", "Sold total", "Avg. price" ],
        Array [ "foo1", 200, 200, 0, 0, 1 ],
      ]
```

Reply:
You're absolutely right! This output is Jest's default and I'm not sure if Prettier also plays a part on separating items per line, but will look into it (and just to be extra clear and probably redundant, the inline snapshot is auto-generated).